doc/_templates/respirator/src/utils.rst.j2

```diff
@@ -0,0 +1,4 @@
+{% extends "toc.rst.j2" %}
+{% block description %}
+lorem ipsum
```

Comment:
looks generic. Intended?

Reply:
This has been replaced with a proper description